tests/chainer_tests/serializers_tests/test_npz.py

```diff
@@ -252,8 +255,8 @@ def _check_optimizer_group(self, file, state, prefix=''):
                 'child/linear/b/msdx',
                 'child/Wc/msg',
                 'child/Wc/msdx') + state
-        self.assertSetEqual(set(file.keys()),
-                            {prefix + x for x in keys})
+        self.assertEqual(set(file.keys()),
+                         {prefix + x for x in keys})
```

Comment:
I see. If we should change `v1`, too. It is better to make another PR whose destination is the master branch.